tools/donate_cpu_lib.py

```diff
@@ -16,7 +16,7 @@
 # Version scheme (MAJOR.MINOR.PATCH) should orientate on "Semantic Versioning" https://semver.org/
 # Every change in this script should result in increasing the version number accordingly (exceptions may be cosmetic
 # changes)
-CLIENT_VERSION = "1.4.0"
+CLIENT_VERSION = "1.3.52"
```

Comment:
Please set it to `1.3.53`.

As `1.3.52` was attached to multiple versions it would make it impossible to tell if the latest version was actually used.